src/Compilers/CSharp/Portable/Operations/CSharpOperationFactory.cs

```diff
@@ -280,14 +280,25 @@ private IPlaceholderExpression CreateBoundDeconstructValuePlaceholderOperation(B
             return new PlaceholderExpression(_semanticModel, syntax, type, constantValue, isImplicit);
         }
 
-        private IInvocationExpression CreateBoundCallOperation(BoundCall boundCall)
+        private IOperation CreateBoundCallOperation(BoundCall boundCall)
         {
-            IMethodSymbol targetMethod = boundCall.Method;
+            MethodSymbol targetMethod = boundCall.Method;
+            SyntaxNode syntax = boundCall.Syntax;
+            ITypeSymbol type = boundCall.Type;
+            Optional<object> constantValue = ConvertToOptional(boundCall.ConstantValue);
+            bool isImplicit = boundCall.WasCompilerGenerated;
+            
+            if (!boundCall.OriginalMethodsOpt.IsDefault || boundCall.ResultKind == LookupResultKind.OverloadResolutionFailure || targetMethod == null || targetMethod.OriginalDefinition is ErrorMethodSymbol)
```

Comment:
@AlekseyTs I can't figure out how to hit the `targetMethod == null || targetMethod.OriginalDefinition is ErrorMethodSymbol` part of the condition.